src/utils/event-utils.ts

```diff
@@ -32,6 +32,16 @@ export const CAMPAIGN_PARAMS = [
 ]
 
 export const Info = {
```

Comment:
I know it's not this PR but "info" makes me sad 🤣 
we should rename it "misc" :p